### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by fncischen**
@ryanboris Resume Website - PreProcessing HTML.

### CODE REVIEW

**Review by secantsquared — APPROVED** [review]
@fncischen 
O M G
WOW

Without even the content fully in there and even thought is still somewhat in the skeleton phase this layout...

wow

totally memorable, aesthetic and impressive
wow

File: less/index.less [hunk 1]
```diff
@@ -54,6 +54,127 @@ table {
...
+				
+				li {
+					list-style-type: none;
+					margin: 1%;
+					padding: 2%;
+					
+					a {
+						border: @borderred;
+						text-decoration: none;
+						padding: 15%;
```
Comment (secantsquared) [inline less/index.less]:
```suggestion
						padding: 15%;
```

i think this padding may be messing up the a tags when the view port changes.

maybe try 
```css
padding: 10px 5%
```

**Review by secantsquared — CHANGES_REQUESTED** [review]
Take a look at the fluidity of the nav as is gets smaller - overall great job

File: less/index.less (hunk 1 above)
Comment (fncischen) [reply less/index.less]: I got a chance to take a look at this this evening - still having the same issues, after adding the padding! Would love to hop on a zoom call later this coming afternoon to clarify.

**Review by secantsquared — APPROVED** [review]

**Pull request merged**